packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,133 @@
+import { css } from '@microsoft/fast-element';
+import { forcedColorsStylesheetBehavior } from '@microsoft/fast-foundation';
+import {
+  accentFillActiveBehavior,
+  accentFillHoverBehavior,
+  accentFillRestBehavior,
+  neutralFillHoverBehavior,
+  neutralFillInputActiveBehavior,
+  neutralFillInputHoverBehavior,
+  neutralFillInputRestBehavior,
+  neutralFillRestBehavior,
+  neutralFocusBehavior,
+  neutralForegroundRestBehavior,
+  neutralOutlineRestBehavior,
+} from '../styles/index';
+
+export const TooltipStyles = css`
+  :host {
+    position: relative;
+    contain: layout;
+    overflow: visible;
+    height: 0;
+    width: 0;
+    z-index: 10000;
+  }
+
+  .tooltip {
+    box-sizing: border-box;
+    border-radius: calc(var(--corner-radius) * 1px);
+    border: calc(var(--outline-width) * 1px) solid transparent;
+    background: ${neutralForegroundRestBehavior.var};
+    color: ${neutralFillRestBehavior.var};
```

Comment:
Should we invert these recipes as you've done here or should we have a separate recipe for this? My concern is what happens on different surfaces such as L1, cards, etc...will this do what we expect or will it respond incorrectly due to fundamental differences in the requirements (how we push to meet contrast)?

Reply:
You bring up a good point and we should have a new recipe. I was missing elevation on it, which is to help distinguish the tooltip with a darker background. But in the case of both the background and the tooltip background, being the same color, elevation will barely help. So I will work with Brian on getting a recipe for the color in the design. 
![image](https://user-images.githubusercontent.com/37851220/105782751-ee39df00-5f29-11eb-9109-613adb6ebdf9.png)

Reply:
updated to use the neutral contrast recipe